probe.go

```diff
@@ -191,6 +191,107 @@ func probeSystemVDOMResources(c FortiHTTP, registry *prometheus.Registry) bool {
 	return true
 }
 
+func probeVPNStatistics(c FortiHTTP, registry *prometheus.Registry) bool {
+	var (
+		vpncon = prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: "fortigate_vpn_connections_count_total",
+				Help: "Number of VPN connections",
+			},
+			[]string{"vdom"},
+		)
+	)
+	registry.MustRegister(vpncon)
+	type result struct {
+		Results []map[string]interface{}
+		VDOM    string
+	}
+	var res []result
+	if err := c.Get("api/v2/monitor/vpn/ssl", "vdom=*", &res); err != nil {
+		log.Printf("Error: %v", err)
+		return false
+	}
+
+	for _, v := range res {
+		count := len(v.Results)
+		vpncon.WithLabelValues(v.VDOM).Set(float64(count))
+	}
+
+	return true
+
+
+}
+func probeIPSec(c FortiHTTP, registry *prometheus.Registry) bool {
+	var (
+		status = prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{ // change type?
```

Comment:
This should be a gauge as far as I can tell, so the comment might be ok to remove?